asserts/pool.go

```diff
@@ -273,6 +368,22 @@ func (p *Pool) phase(ph poolPhase) error {
 	return nil
 }
 
+func (p *Pool) AddSequenceToUpdate(unresolved *AtSequence, group string) error {
+	if err := p.phase(poolPhaseAddUnresolved); err != nil {
+		return err
+	}
+	if unresolved.Pinned && unresolved.Sequence <= 0 {
```

Comment:
actually I don't think it makes sense to use pass something with Sequence not set to Update? it wouldn't be an update otherwise? which maybe helps clarify what isResolvedSeq should do?